app/domain/logs.rb

```diff
@@ -591,106 +591,106 @@ module AuthnJwt
       )
 
       ClaimsDenyListValue = ::Util::TrackableLogMessageClass.new(
-        msg: "Claims denylist value is '{0-deny-claims-list}'",
+        msg:  "Claims denylist value is '{0-deny-claims-list}'",
         code: "CONJ00117D"
       )
 
       ValidatingClaimName = ::Util::TrackableLogMessageClass.new(
-        msg: "Validating claim name '{0-claim-name}'",
+        msg:  "Validating claim name '{0-claim-name}'",
         code: "CONJ00118D"
       )
 
       ValidatedClaimName = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully validated claim name '{0-claim-name}'",
+        msg:  "Successfully validated claim name '{0-claim-name}'",
         code: "CONJ00119D"
       )
       ParsingMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Parsing mandatory claims value '{0-mandatory-claims}'",
+        msg:  "Parsing mandatory claims value '{0-mandatory-claims}'",
         code: "CONJ00120D"
       )
 
       ParsedMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully parsed mandatory claims '{0-mandatory-claims-list}'",
+        msg:  "Successfully parsed mandatory claims '{0-mandatory-claims-list}'",
         code: "CONJ00121D"
       )
 
       FetchingMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching mandatory claims...",
+        msg:  "Fetching mandatory claims...",
         code: "CONJ00122D"
       )
 
       NotConfiguredMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Mandatory claims is not configured",
+        msg:  "Mandatory claims is not configured",
```

Comment:
what does this mean?

Reply:
It means that `enforced-claims` variable is not defined in authenticator's policy

Reply:
mandatory has been renamed to enforced